website/docs/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -191,6 +223,14 @@ The "read" permission set is for people who need to view information about the s
 - Read the project name.
 - Read the workspaces in the project.
 
+### Custom Project Permissions
+
+Custom permissions let you assign specific, finer-grained permissions to a team than the broader fixed permission sets provide. This enables more task-focused permission sets and tighter control of sensitive information.
+
+You can use custom permissions to assign any of the permissions listed above under [General Project Permissions](#general-project-permissions).
+
+Some permissions - such as the project access permission - are tiered: you can assign one permission per category, since higher permissions include all of the capabilities of the lower ones.
```

Comment:
```suggestion
Some permissions, such as the project access permission, are tiered. You can only assign one permission per category because higher-level permissions include the capabilities of lower levels.
```